vulkan/vulkan.hpp

```diff
@@ -454,8 +469,8 @@ namespace vk
 
     StructureChain& operator=(StructureChain const &rhs)
     {

```

Comment:
This change is not expected. Did you generate with header with the latest version of Vulkan-Hpp including your changes?

Reply:
Yes, I just did it using the exact code in this PR.